zilencer/lib/stripe.py

```diff
@@ -122,10 +130,10 @@ def do_create_customer_with_payment_source(user: UserProfile, stripe_token: str)
     return stripe_customer
 
 @catch_stripe_errors
-def do_subscribe_customer_to_plan(stripe_customer: stripe.Customer, stripe_plan_id: int,
+def do_subscribe_customer_to_plan(stripe_customer: stripe.Customer, stripe_plan_id: str,
                                   seat_count: int, tax_percent: float) -> None:
-    # TODO: check that there are no existing live Stripe subscriptions
-    # (canceled subscriptions are ok)
+    if extract_current_subscription(stripe_customer) is not None:
+        raise AssertionError("Customer already has a subscription.")
```

Comment:
Let's log this as an error and send the generic error message 
"Something went wrong. Please contact support@.." 
to the frontend?

Reply:
Though we should make sure we figure out how to test these situations, given that all the error messages will be the same. Is there some meta-data that can be added to the Exception that the testing suite would be able to check against?

Reply:
I will create a PR for catching all the errors separately.